docs/app/Component Guidelines.md

```diff
@@ -74,16 +85,68 @@ export default class Input extends Component {
 ```
 >Never wrap the component with other components, whether DOM or Composite.
 
-## Has props.className after `sd-<component>`
+## Events
+
+Stardust manages Semantic UI's jQuery via React events and lifecycle methods.
+
+Example, the Message component can be dismissed on click of the close icon. Per
+the Semantic UI docs, this is done by calling Semantic UI's `transition()`
+jQuery plugin on the message via a jQuery click event handler. Instead, 
+Stardust uses React's `onClick` listener to trigger the `transition()`.
```

Comment:
This "Events" section is the new content.  All other changes were clarifications, typo fixes, and reordering.